docs/src/suitable_commands.md

```diff
@@ -0,0 +1,55 @@
+# Suitable commands
+
+### help
+
+### index
+
+- [x] Manage(list, add, delete) custom plugin indexes
+
+### info
+
+- [x] Show information about a plugin via NAME & INDEX/NAME
+
+### install - Downloads, Extract & Install a plugin
+
+- [x] Install one NAME or multiple plugins [Name, Name] from default index
+- [ ] Install plugins from a newline-delimited txt file
+- [x] Install one or mutiple plugins from custom index
+- [ ] Install plugin from a local/URL custom manifest file
+- [x] If a plugin is already installed, it will be skipped
+- [x] Failiure to install a plugin will not stop the installation of other plugins
+- [ ] Cleanup Stale Installations removes the versions that aren't the current version
+
+### list
+
+- [x] show list of all charmil plugins
```

Comment:
There is no concept of charmil plugins - Charmil is framework that let others to build their plugin indexes and embed them into their CLI. We need to be clear about this difference - I think I prematurely closed #7

Reply:
now it's all clear.